build.gradle

```diff
@@ -28,5 +32,61 @@ wrapper {
 }
 
 test {
```

Comment:
I'm trying to understand this 😅  Why is this block inside build.gradle?

Reply:
The test block allows you to customize the execution of the tests, it was built for ease of testing and with future-oriented execution under IC.

In this case, according to the regressionType parameter, it is determined which tests should be executed